spec/cell_spec.rb

```diff
@@ -0,0 +1,184 @@
+require 'rspec'
+require 'pry'
+require './lib/cell'
+require './lib/ship'
+
+# !!! We need to add a test if an invalid cell is given in the future.!!!
+
+RSpec.describe Cell do
+  describe '#initialize' do
+    it 'exists' do
```

Comment:
This is a just a nit: tests are a bit more readable with white space between the `it` blocks